_data/terms-of-service.yml

```diff
@@ -222,7 +230,7 @@
 
 - category_name: Gruntwork Subscription Terms
   category_id: gruntwork-subscription
-  category_description: This attachment was last updated on May 8, 2019. Unless otherwise defined in these Gruntwork Subscription Terms, capitalized terms will have the meaning given to them in the Terms.
+  category_description: This attachment was last updated on February XX, 2020. Unless otherwise defined in these Gruntwork Subscription Terms, capitalized terms will have the meaning given to them in the Terms.
```

Comment:
Fill in actual date.

Reply:
@josh-padnick If you see the PR body; I was asking a few questions including the date you'd like to appear here.

Reply:
Ah, sorry I missed that! You can use today's date: Feb 7, 2020.